tests/setup-runtime-dependencies.cmd

```diff
@@ -42,22 +42,29 @@ REM ============================================================================
 set __DotNetToolDir=%__ThisScriptPath%..\Tools
 set __DotNetCmd=%__DotNetToolDir%\dotnetcli\bin\dotnet.exe
 set __PackageDir=%__ThisScriptPath%..\Packages
-set __JasonFilePath=%__ThisScriptPath%project.json
+set __TmpDir=%Tmp%\coreclr_gcstress_%RANDOM%
```

Comment:
We should always use %TEMP%. It does seem that %TMP% is also defined, but I think for consistency we should use TEMP. (TMP is there for backwards-compatibility)